src/game/models/asteroid.rs

```diff
@@ -9,11 +20,14 @@ use rand;
 use super::super::color;
 use super::{Collidable, Drawable, PI_MULT_2, Positioned, Updateable, Vector};
 
+/// The number of segments in an asteroid's shape is currently set at compile-time
 const NUM_SEGMENTS: usize = 20;
+/// Asteroids have random radiuses within the defined range
 const RADIUS_MIN: f64 = 15.0;
 const RADIUS_MAX: f64 = 70.0;
+/// Asteroids shapes are made by mutating a circle, this is a magic number to tune that.
```

Comment:
I think this section would read better if there was a newline preceding the doc comments, so it's more clear which item the docs are for.